ui/admin/tests/unit/controllers/scopes/scope/groups/group/index-test.js

```diff
@@ -0,0 +1,13 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+
+module('Unit | Controller | scopes/scope/groups/group/index', function (hooks) {
+  setupTest(hooks);
+
+  test('it exists', function (assert) {
```

Comment:
None blocking comment: I just realize that we do not have any real controller test in our codebase. We are just testing that the controller exists. I will bring this topic to tech time since I think is a big one and needs team members' opinion on it.

Reply:
good point, and thank you!

Reply:
Agreed that we should be testing our controllers, especially now that we are moving our actions from the routes to the controllers. We have some example [tests for controllers](https://github.com/hashicorp/boundary-ui/blob/main/ui/admin/tests/unit/controllers/scopes/scope/session-recordings/session-recording/channels-by-connection/index-test.js) but they seem to be mostly for getters which are a bit easier to test.

Reply:
For actions added to the controller in this PR, I'll make tickets to add tests. For future work, ill add the tests in the same PR.

Reply:
@lisbet-alvarez Amazing!! yeah, creating tickets to bring tests later is ok. In this case we do not have patterns to follow so all good providing separate PR's for those so we can focus on them and bring discussions.